docs/configuration/index.md

```diff
@@ -1433,7 +1433,7 @@ Druid uses Jetty to serve HTTP requests.
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`druid.processing.buffer.sizeBytes`|This specifies a buffer size (less than 2GiB) for the storage of intermediate results. The computation engine in the Indexer processes will use a scratch buffer of this size to do all of their intermediate computations off-heap. Larger values allow for more aggregations in a single pass over the data while smaller values can require more passes depending on the query that is being executed. [Human-readable format](human-readable-byte.md) is supported.|auto (max 1GB)|
```

Comment:
max value is `1GiB` according to the code

https://github.com/apache/druid/blob/0c5d1c9725303111df69198a232166ddf00672d2/processing/src/main/java/org/apache/druid/query/DruidProcessingConfig.java#L39